test/v3/addLiquidity/addLiquidity.integration.test.ts

```diff
@@ -438,6 +438,7 @@ describe('add liquidity test', () => {
                 );
             });
         });
+        // FIXME: test proportional case on pool with >18 decimal tokens - it will fail
```

Comment:
Can we open an issue for this?

Reply:
That not actually needed - I initially thought about testing the proportional helper by testing the add liquidity flow, but in the end, it made more sense to test the helper directly.
I'll remove the comment